src/coreclr/interpreter/intops.def

```diff
@@ -31,6 +31,22 @@ OPDEF(INTOP_MOV_4, "mov.4", 3, 1, 1, InterpOpNoArgs)
 OPDEF(INTOP_MOV_8, "mov.8", 3, 1, 1, InterpOpNoArgs)
 OPDEF(INTOP_MOV_VT, "mov.vt", 4, 1, 1, InterpOpInt)
 
+OPDEF(INTOP_LOAD_I4_I1, "load.i4.i1", 3, 1, 1, InterpOpNoArgs)
```

Comment:
As with previous discussions on this topic, sounds to me that this frame var indirection is only needed for filter funclets. In this case it doesn't make sense to have so many opcodes, we could replace all of them via a single `load.framevar` and already existing indirect loads.

Reply:
I still recall that discussion, although I am not sure if reducing the number of opcodes should be a goal. I think we might likely increase the time spent on executing more opcodes and also making the code a bit more complicated.

I will give that a try to confirm if that change worth it.